contracts/RiskManagerV1.sol

```diff
@@ -214,6 +214,24 @@ contract RiskManagerV1 is Auctioneer, Ownable {
         signerBondsSwapStrategy.swapSignerBonds{value: withdrawableAmount}();
     }
 
+    /// @notice Throws if deposit not present among deposits in liquidation or
+    // the state of the deposit is not DEPOSIT_LIQUIDATION_IN_PROGRESS_STATE.
+    /// @dev This function is called when the auctioneer is informed about the
+    /// results of an auction. It is invoked from the Auctioneer contract's
+    /// offerTaken function.
+    /// @param auction Address of an auction whose deposit needs to be checked.
+    function checkDepositState(address auction) internal override {
+        require(
```

Comment:
I'm not sure if this check is needed here.  `checkDepositState()` can be called only internally through `takeOffer()` on an _opened_ auction. If an auction is closed (deleted by `harikari()`), we won't be able to call `takeOffer()` in the first place. I think it is enough to check the `deposit.currentState()` in case a deposit was bought outside the coverage pool.

Reply:
I added this check just to be on the safe side.
But, indeed, it is unnecessary (we cannot enter this function for a close auction).
Removed in  50997b5081f5d421e73bc66555abb64f14dbd672.